workflows/zScanAction.yml

```diff
@@ -1,95 +1,60 @@
+# and identifies security, privacy, and compliance-related vulnerabilities. ​
+#
+# Prerequisites:
+#  * An active Zimperium zScan account is required. If you are not an existing Zimperium
+#    zScan customer, please request a zSCAN demo by visiting https://www.zimperium.com/contact-us.
+#  * Either GitHub Advanced Security (GHAS) or a public repository is required to display
+#    issues and view the remediation information inside of GitHub code scanning alerts.​
+#
+# For additional information and setup instructions
+# please visit:  https://github.com/Zimperium/zScanMarketplace#readme
+
+name: "Zimperium zScan"
 
 on:
   push:
-    branches: [master]
+    branches: [ "main" ]
   pull_request:
-    branches: [master]
-  workflow_dispatch:
-    branches: [master]
-    
-jobs:
-  build:
-    strategy:
-      matrix:
-        os: [ubuntu-latest]
-    runs-on: ${{ matrix.os }}
-    steps:
-    - uses: actions/checkout@v3
-    - uses: actions/setup-java@v3
-      with:
-        distribution: temurin
-        java-version: 11
-        
-    - name: Setup Gradle
-      uses: gradle/gradle-build-action@v2
-      with:
-        gradle-version: 7.3.3
-    
-    - name: Execute Gradle build
-      run: ./gradlew build
+    branches: [ "main" ]
 
-    - name: Get apk path
-      id: apk-path
-      run: |
-        path=$(find **/build/outputs/apk -name '*.apk' -type f | head -1)
-        echo "::set-output name=path::$path"
-        
-    - name: Get apk info
-      id: apk-info
-      uses: hkusu/apk-info-action@v1
-      with:
-        apk-path: ${{ steps.apk-path.outputs.path }}
+permissions:
+  contents: read
 
-    - name: Move Artifact
-      uses: actions/upload-artifact@v3
-      with:
-        name: my-artifact
-        path: ${{ steps.apk-path.outputs.path }}
-        if-no-files-found: error # 'warn' or 'ignore' are also available, defaults to `warn`
-  
-  scan:
-    strategy:
-      matrix:
-        os: [ubuntu-latest]
-    runs-on: ${{ matrix.os }}
-    needs: build
+jobs:
+  zscan:
+    name: zScan
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read          # for actions/checkout to fetch code
+      security-events: write  # for github/codeql-action/upload-sarif to upload SARIF results
+      actions: read           # only required for a private repository by github/codeql-action/upload-sarif to get the Action run status
     steps:
-    - name: Checkout repository
-      uses: actions/checkout@v2
-
-    - name: Download application
-      uses: actions/download-artifact@v2
-      with:
-        name: my-artifact
-
-    - name: Zimperium
-      uses: zimperium/zscanmarketplace@v1
-      timeout-minutes: 60
-      with:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+  #  - name: Execute gradle build
+  #    run: echo "./gradlew build"
+      
+      - name: Run Zimperium zScan
+        uses: zimperium/zscanmarketplace@bfc6670f6648d796098c251ccefcfdb98983174d
```

Comment:
The problem with linking to an exact version of the zscan marketplace action is if we fix a bug in the action then there will be a new jumble of number/letters and anyone using the action will have to manually go change their build.   I know github recommends this as a best practice, but it doesn't seem great for bug fix releases.